smoke/tests/tool/nydusd.go

```diff
@@ -622,17 +645,20 @@ func (nydusd *Nydusd) GetInflightMetrics() (*InflightMetrics, error) {
 }
 
 func (nydusd *Nydusd) Verify(t *testing.T, expectedFileTree map[string]*File) {
+	nydusd.VerifyByPath(t, expectedFileTree, nydusd.MountPath)
```

Comment:
Sorry for the misunderstand, we should call `nydusd.VerifyByPath(t, expectedFileTree, "")` here.